superset-frontend/src/utils/common.js

```diff
@@ -66,10 +66,10 @@ export function optionLabel(opt) {
     return EMPTY_STRING;
   }
   if (opt === true) {
-    return '<true>';
+    return 'true';
```

Comment:
There's a `TRUE_STRING` and `FALSE_STRING` constants. Can we use them?

Reply:
Done! Though now we'll have `TRUE` instead of `true` in the labels